app/screens/home/home_screen.tsx

```diff
@@ -47,21 +48,26 @@ export default function HomeScreen({ user, navigation, firestoreCtrl }: any) {
         contentContainerStyle={styles.contentContainer}
         colorType="transparent"
       >
-        {challenges.map((challenge, index) => (
-          <Challenge
-            navigation={navigation}
-            firestoreCtrl={firestoreCtrl}
-            key={index}
-            challengeDB={challenge}
-            // Include other props as needed
-          />
-        ))}
+        {challenges.length === 0 ? (
+          <ThemedText>No challenge to display</ThemedText>
+        ) : (
+          challenges.map((challenge, index) => (
+            <Challenge
+              navigation={navigation}
+              firestoreCtrl={firestoreCtrl}
```

Comment:
This should include a testID for testing

Reply:
Despite what GitHub implies, I'm not responsible for the Challenge component. This should be addressed in a different PR.